routes/financial/testIncome.json

```diff
@@ -0,0 +1,30 @@
+{
+  "income" : {
+    "employer 1": {
+      "total": 11000.00
+    },
+    "employer 2": {
+      "total": 17000.00
+    }
+  },
+  "totalIncome" : 28000.00,
+  "taxes" : {
+    "federal" : {
+      "name" : "Federal tax",
+      "amount" : 1998.00
+    },
+    "provincial" : {
+      "name" : "Provincial tax",
+      "amount" : 1106.00
+    },
+    "CPP" : {
+      "name" : "CPP",
+      "amount" : 1213.00
+    },
+    "EI" : {
+      "name" : "Provincial tax",
+      "amount" : 456.00
+    }
+  },
+  "totalTax": 4782.00
+}
```

Comment:
Do you want to merge this into the `user.json`, or was there a reason not to (other than that I suggested you hardcode it and don't put it in the user.json file)? 👼 

Also, I was taking the "super easy to implement" approach of keeping income as strings so that we don't have to format. (https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/api/user.json#L23). No preference, really.

Reply:
I'm not really bothered right now whether we keep this here or not (not really a blocker for testing), but we shouldn't put the benefits review content in a new file too.

I think, at this point, that this could just be part of the user info.